css/components/_verse.css

```diff
@@ -9,7 +9,7 @@
 	width: 100%;
```

Comment:
This width being set 100% breaks the layout on Desktop, but that's an fix in another PR if needed.